aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,134 @@
+
+Wait for Visual Studio to create the project and for the project's dependencies to be restored:
+
+:::image type="content" source="maui/_static/restored-dependencies.png" alt-text="Restored dependencies.":::
+
+In the **Android SDK License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/android-sdk-license.png" alt-text="Android SDK License Acceptance window.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+Wait for Visual Studio to download the Android SDK and Android Emulator.
+
+## Run the app in the Windows Machine Emulator
+
+In the Visual Studio toolbar, select the **Windows Machine** button to build the app:
+
+:::image type="content" source="maui/_static/windows-machine-button.png" alt-text="Windows Machine button.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+If Developer Mode isn't enabled, you're prompted to enable it in **Settings** > **For developers** > **Developer Mode**. Set the switch to **On**:
+
+:::image type="content" source="maui/_static/windows-developer-mode.png" alt-text="Windows Developer Mode enabled.":::
+
+The running app in the Windows Machine Emulator:
+
+:::image type="content" source="maui/_static/running-app-windows-1.png" alt-text="App running in the Windows Machine Emulator.":::
+
+Navigate to the **`Counter`** page. Select the **`Click me`** button to increment the counter:
+
+:::image type="content" source="maui/_static/running-app-windows-2.png" alt-text="Counter page running in the Windows Machine Emulator.":::
+
+## Run the app in the Android Emulator
+
+In the Visual Studio toolbar, select the **Android Emulator** button to build the app:
+
+:::image type="content" source="maui/_static/android-emulator-button.png" alt-text="Android Emulator button.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+In the **New Device** window, select the **Create** button:
+
+:::image type="content" source="maui/_static/new-android-device.png" alt-text="New Android Device window.":::
+
+In the **License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/license-acceptance.png" alt-text="License Acceptance window.":::
+
+Wait for Visual Studio to download, unzip, and create an Android Emulator.
+
+Close the **Android Device Manager** window:
+
+:::image type="content" source="maui/_static/android-device-manager.png" alt-text="Android Device Manager window.":::
+
+In the Visual Studio toolbar, select the **Pixel 5 - API 30 (Android 11.0 - API 30)** button to build and run the app:
```

Comment:
Presumably the Android version will change. For now this is fine, but it would be nice if we could somehow insulate this tutorial from churn due to newer Android versions.